tests/Nightly/test_neural_speed.py

```diff
@@ -95,6 +95,14 @@ def test_gguf_api(self):
         assert(output[0][0:15] == [1, 5713, 3714, 264, 727, 28725, 736, 403, 264, 1628, 2746, 693, 6045, 298, 1220])
 
 
+    def test_use_vllm_api(self):
+        model_name = "/tf_dataset2/models/nlp_toolkit/llama-2-7b-chat/Llama-2-7b-chat-hf"
+        prompt = "Once upon a time"
+        model = AutoModelForCausalLM.from_pretrained(model_name, use_vllm = True)
```

Comment:
better to add a new ut script, rather than use neural speed one.

Reply:
fixed